src/preferences/colorpalettesettings.cpp

```diff
@@ -0,0 +1,106 @@
+#include "preferences/colorpalettesettings.h"
+
+namespace {
+const mixxx::RgbColor kColorBlack(0x000000);
+const QString kColorPaletteConfigGroup = QStringLiteral("[Config]");
+const QString kColorPaletteGroup = QStringLiteral("[ColorPalette %1]");
+const QRegExp kColorPaletteGroupNameRegex("^\\[ColorPalette (.+)\\]$");
```

Comment:
QStringLiteral can be used here as well.
Or even better to replace the regex with https://doc.qt.io/qt-5/qstring.html#startsWith

Reply:
The regex also makes sure to only return color palette names with non-empty names, and we do matching and stripping out the name in the same step. Are regexes really that much slower?

Reply:
Yes, they are way sower. I was not aware that it does more things at once, so we can keep it here.